### ISSUE TITLE
[1.12.2] Discussion: Alumite stats and traits

### ISSUE THREAD

**Issue by TurboPig** [TurboPig/Tinkegration#3]
Now that alumite is implemented, I am opening a discussion on the details of its implementation.

**Mining level**
As of version 0.2, alumite has the mining level "Cobalt."  However, I have considered reducing it to "Obsidian" for the following reasons:
1. In 1.7.10, alumite was ["on par"](http://tinkers-construct.wikia.com/wiki/Alumite) with steel.  At the time, both steel and alumite had a mining level of "Cobalt."  However, in 1.12.2, steel now has a mining level of "Obsidian."
2. It is relatively easy to obtain alumite, so reducing its mining level could be more balanced?  _That being said,_ because it is an alloy of obsidian, which has a mining level of "Cobalt," it is arguable that the mining level should match it.

**Traits**
Not going to lie - I couldn't figure out an appropriate trait for alumite.  It currently has "Tempered II" and "Tempered," just like invar.  I'm not sure what kind of trait effect would make sense for alumite.

**Comment by carstorm** [comment]
Just wondering if your aware that the mod [PlusTic ](https://minecraft.curseforge.com/projects/plustic) has Alumite which adds the trait Global Traveler? Belive this would cause conflicts since its the same name?

EDIT: I should of read the description fully before commenting. It seems like you may be aware of this. IMO due to what Alumite does in PlusTic, it is far supior. As it description states "Global Traveler - dropped items can transport to inventory previously shift-right-clicked".

**Comment by TurboPig** [comment]
True, PlusTiC's trait for alumite is objectively superior.  However, I personally don't think it makes much sense; why does alumite have ender-like capabilities?  It is an alloy of terrestrial materials.  Hence, I started developing this mod :P

**Comment by Preminenc3** [comment]
Sorry for the late reply but I think reducing it down to obsidian would make more sense. Technically when you mix different ores you don't always receive the benefits of all of them so lowering it to obsidian, lowering durability, and increasing the speed would be a good balance.
When would the next update for the mod be? need any help coming up with more traits then I can help out; I posted a PDF that had about maybe 20 traits.

**Comment by TurboPig** [comment]
Howdy, I'm not sure when the next version will be; I'm having trouble with Botania support, and school has been keeping me a bit busy lately.

Any trait suggestions would be great :)
